README.md

```diff
@@ -1 +1,62 @@
 # To-Do-List
+
+> To Do List, creating modules on javascript using webpack, localStorage and adding dynamic event listeners.
+
+![screenshot]()
+
+The main goal of this project is understanding of the benefits of modularized code as well as the ability to set up a javascript application made of different modules using webpack and localStorage to persist the website data.
+
+## Built With
+
+- HTML
+- SCSS
+- Bootstrap
+- JavaScript
+- Webpack
+- LocalStorage
+
+## Live Demo
+
+[Live Demo Link]()
+
+## Getting Started
+
+- Clone this project on your computer.
+- Open the files on you text editor.
+- Open the .html on your browser to see the future changes.
+
+### Prerequisites
+
+- You have to get a text editor to inspect the code in your computer.
+
+### Install
+
+- Install npm and webpack commands.
```

Comment:
Webpack and family should be listed on your devDependencies, so it should be install when you run `npm install` to install.

Reply:
Please see this

Reply:
fixed